### PR TITLE
update rsj.sty and make with latexmk

### PR DESCRIPTION
**Pull request by knorth55**
- update `rsj.sty` to current sty file
- make with `latexmk` (like p-report)
- add travis

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,35 @@
-# rsj-report
+# rsj-template
```
Comment (pazeshun) [inline README.md]: This repository's name is `rsj-report`, not `rsj-template`

File: README.md [hunk 2]
```diff
@@ -1 +1,35 @@
...
+[![Build Status](https://travis-ci.com/jsk-report-template/rsj-report.svg?branch=master)](https://travis-ci.com/github/jsk-report-template/rsj-report)
+
+Latex template for RSJ
+
+### 1. Prerequisities
+
+```bash
+sudo apt-get update
+make install
```
Comment (pazeshun) [inline README.md]: Same as https://github.com/jsk-report-template/p-report/pull/6#discussion_r457986180

File: README.md [hunk 3]
```diff
@@ -1 +1,35 @@
...
+```bash
+make
+# or
+latexmk -pvc main
+```
+
+### 4. Make publication pdf 
+
+```bash
+make pub
```
Comment (pazeshun) [inline README.md:19]: Same as https://github.com/jsk-report-template/p-report/pull/6#discussion_r457999966

**Comment by knorth55** [comment]
@pazeshun can you review this PR?
I switch to use p-report like format and build system.

**Review by pazeshun — CHANGES_REQUESTED** [review]

**Comment by knorth55** [comment]
@pazeshun thanks, I updated this PR.

File: README.md [hunk 4]
```diff
@@ -1 +1,32 @@
...
+```bash
+make
+# or
+latexmk -pvc main
+```
+
+### Optional. Convert Japanese punctuations
+
+```bash
+$ make pub
```
Comment (pazeshun) [inline README.md]: Same as https://github.com/jsk-report-template/p-report/pull/6#discussion_r458103947

File: README.md [hunk 5]
```diff
@@ -1 +1,32 @@
...
+# or
+$ make publish
```
Comment (pazeshun) [inline README.md]: https://github.com/jsk-report-template/p-report/pull/6#discussion_r458103947

**Review by pazeshun — CHANGES_REQUESTED** [review]

**Comment by knorth55** [comment]
@pazeshun thanks, I updated.

**Review by pazeshun — APPROVED** [review]

**Pull request merged**